tools/check_circleci_config.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+"""
+Utility to ensure correctness of the CircleCI workflow
+"""
+import sys
+from difflib import unified_diff
+
+import click
+import yaml
+from yaml.parser import ParserError
+
+
+def _red(string):
+    return click.style(string, fg='red')
+
+
+def _green(string):
+    return click.style(string, fg='green')
+
+
+def _yellow(string):
+    return click.style(string, fg='yellow')
+
+
+def _check_workflows_align(config):
+    """
+    Ensure that the common shared jobs in the `raiden-default` and `nightly` workflows are
```

Comment:
nice idea. Curious question. Is there any occasion where they fell out of sync and prompted you to introduce this script as a check?

Reply:
:+1: